docs/usage/self-hosted-configuration.md

```diff
@@ -128,7 +128,13 @@ e.g.
 
 ## composerIgnorePlatformReqs
 
-Set to `false` to prevent usage of `--ignore-platform-reqs` in the Composer package manager.s
+Set to `false` to prevent usage of `--ignore-platform-reqs` in the Composer package manager.
+
+## composerIgnorePlatformReq
```

Comment:
Can't we combine those options to one ?

Reply:
You mean like making the plural option accept both a bool and an array of strings?

Reply:
Yes. Not sure if it's currently supported.

Reply:
I just checked, that is currently not supported (see https://github.com/renovatebot/renovate/blob/main/lib/config/types.ts#L350).

Reply:
New idea:

- migrate `composerIgnorePlatformReqs=false` to `composerIgnorePlatformReqs=null`.
- if `composerIgnorePlatformReqs=[]` add `--ignore-platform-reqs` to command
- if `composerIgnorePlatformReqs` is nonempty string array, add individual `--ignore-platform-req xxx` options

Reply:
That actually sounds like a great solution! I will however wait for @rarkins to share his opinion before updating the implementation 😄

Reply:
I had an empty moment, so I implemented it anyways. It already includes to configuration migration, so it should be ready to go 👍🏻